src/ILCompiler/src/Program.cs

```diff
@@ -252,7 +252,7 @@ private int Run(string[] args)
                 else if (_targetArchitectureStr.Equals("arm", StringComparison.OrdinalIgnoreCase))
                     _targetArchitecture = TargetArchitecture.ARM;
                 else if (_targetArchitectureStr.Equals("armel", StringComparison.OrdinalIgnoreCase))
-                    _targetArchitecture = TargetArchitecture.ARMEL;
+                    _targetArchitecture = TargetArchitecture.ARM;
```

Comment:
Do we still need the command line argument if we treat this same as `arm`?

Reply:
I'm hesitant to remove it because `--targetarch` option might come from running tests (which are built for armel), and then we'll hit the unsupported architecture exception.